lib/Basics/sdt.h

```diff
@@ -0,0 +1,425 @@
+/* <sys/sdt.h> - Systemtap static probe definition macros.
```

Comment:
Why do we bring our own instead of testing with `__has_include`?

Reply:
It is a single header. As far as I see it does not exist in alpine Linux. Therefore I chose this approach. The code is in the public domain. It should even be processor independent and might even work on Mac, no idea, though.

Reply:
We could just compile for a local PC to do some testing? Do we really need alpine support?